### PR TITLE
gofmt and goimports to format code

### PR DESCRIPTION
**Pull request by caiwp**
gofmt  format code

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

File: models/access_test.go [hunk 1]
```diff
@@ -5,8 +5,8 @@
 package models
 
 import (
-	"testing"
 	"github.com/stretchr/testify/assert"
```
Comment (appleboy) [inline models/access_test.go]: move to the bottom of import section.

**Review by appleboy — CHANGES_REQUESTED** [review]

File: models/access_test.go (hunk 1 above)
Comment (caiwp) [reply models/access_test.go]: Just gofmt do

Comment (appleboy) [reply models/access_test.go]: You must be following import path rule like https://github.com/go-gitea/gitea/blob/master/main.go#L9-L16

**Review by appleboy — CHANGES_REQUESTED** [review]

File: models/setup_for_test.go [hunk 2]
```diff
@@ -9,11 +9,12 @@ import (
 	"os"
 	"testing"
 
+	"gopkg.in/testfixtures.v2"
```
Comment (bkcsoft) [inline models/setup_for_test.go:12]: this change is wrong (no matter what goimports says 😆 )

File: modules/base/tool_test.go [hunk 3]
```diff
@@ -3,13 +3,14 @@ package base
 import (
 	"testing"
 
+	"os"
+	"time"
+
 	"code.gitea.io/gitea/modules/setting"
 	"github.com/Unknwon/i18n"
```
Comment (bkcsoft) [inline modules/base/tool_test.go:10]: newline between `code.`- and `github.`-lines :)

File: modules/lfs/server.go [hunk 4]
```diff
@@ -11,11 +11,12 @@ import (
 	"strings"
 	"time"
 
+	"github.com/dgrijalva/jwt-go"
+
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/context"
 	"code.gitea.io/gitea/modules/log"
 	"code.gitea.io/gitea/modules/setting"
-	"github.com/dgrijalva/jwt-go"
```
Comment (bkcsoft) [inline modules/lfs/server.go:18]: same as above, newline between "local" and "external" packages :)

File: modules/templates/helper.go [hunk 5]
```diff
@@ -15,9 +15,10 @@ import (
 	"strings"
 	"time"
 
+	"gopkg.in/editorconfig/editorconfig-core-go.v1"
+
```
Comment (bkcsoft) [inline modules/templates/helper.go:19]: no newline

File: modules/mailer/mailer.go [hunk 6]
```diff
@@ -15,9 +15,10 @@ import (
 	"strings"
 	"time"
 
-	"github.com/jaytaylor/html2text"
 	"gopkg.in/gomail.v2"
 
+	"github.com/jaytaylor/html2text"
```
Comment (bkcsoft) [inline modules/mailer/mailer.go:20]: invalid change

**Review by bkcsoft — CHANGES_REQUESTED** [review]
few things that ticks my OCD 😛

**Comment by bkcsoft** [comment]
Maybe we should document these guidelines?
```go
import (
  "encoding/json"
  "fmt"

  "code.gitea.io/gitea/models"

  "github.com/foo/bar"
)
```

So Stdlib, newline, local packages, newline, external packages. All in alphabetical order of course ^.^

**Comment by appleboy** [comment]
@caiwp Any updates on this?

**Comment by lunny** [comment]
Please rebase.

**Comment by lunny** [comment]
conflicted

File: routers/api/v1/utils/hook.go [hunk 7]
```diff
@@ -7,10 +7,11 @@ package utils
 import (
 	api "code.gitea.io/sdk/gitea"
 
+	"encoding/json"
```
Comment (tboerger) [inline routers/api/v1/utils/hook.go:10]: This should be even above the api import.

**Comment by tboerger** [comment]
Multiple conflicts, any update?

**Comment by caiwp** [comment]
These submitted only problems involving formatted, as long as you are formatted before saving.

**Pull request closed without merging**